Helm/opensearch-dashboards/values.yaml

```diff
@@ -0,0 +1,136 @@
+# Default values for opensearch-dashboards.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+opensearchHosts: "https://opensearch-cluster-master:9200"
+replicaCount: 1
+
+image: "opensearchproject/opensearch-dashboards"
+imageTag: "latest"
+imagePullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+
+rbac:
+  create: true
+
+
+podAnnotations: {}
+
+extraEnvs:
+  - name: "NODE_OPTIONS"
+    value: "--max-old-space-size=1800"
+
+podSecurityContext: {}
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
+
+
+config:
+  dashboards.yml:
+    server:
+        name: dashboards
+        host: 0.0.0.0
+
+    ## Dashboards TLS Config
+        ssl:
+          enabled: true
+          key: /usr/share/dashboards/certs/dashboards-key.pem
+          certificate: /usr/share/dashboards/certs/dashboards-crt.pem
+    opensearch.ssl.certificateAuthorities: /usr/share/dashboards/certs/dashboards-root-ca.pem
+
+priorityClassName: ""
+
+httpPort: 5601
+
+opensearchAccount:
+  secret: ""
+  keyPassphrase:
+    enabled: false
```

Comment:
See previous comments. This seems to assume an existing secret is present? We should be the ones to create this by default:

```
credentials:
  username: xyz
  password: xyz
  key: | 
     <Key>
  keyPassphrase: 123
```

Reply:
Sounds good.